!SESSION 2019-09-28 10:21:47.235 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_144
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.egit.core 1 0 2019-09-28 10:23:15.501
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-09-28 10:23:31.166
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\ITAUser'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.brainwy.liclipsetext.shared_core 4 4 2019-09-28 10:23:32.873
!MESSAGE C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\com.brainwy.liclipse.editor_5.3.0.201908230902\etc\a767gwy0xncwwz0slpfps4tb3 (The system cannot find the path specified)
!STACK 0
java.io.FileNotFoundException: C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\com.brainwy.liclipse.editor_5.3.0.201908230902\etc\a767gwy0xncwwz0slpfps4tb3 (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at org.brainwy.liclipsetext.shared_core.io.FileUtils.writeBytesToFile(FileUtils.java:140)
	at org.brainwy.liclipsetext.shared_core.io.FileUtils.writeStrToFile(FileUtils.java:133)
	at com.brainwy.liclipse.editor.lic.CheckRegistered.run(CheckRegistered.java:146)
!SESSION 2019-09-28 10:23:37.159 -----------------------------------------------
eclipse.buildId=unknown
java.version=1.8.0_144
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Command-line arguments:  -os win32 -ws win32 -arch x86_64

!ENTRY org.eclipse.egit.core 1 0 2019-09-28 10:23:43.035
!MESSAGE Using Apache MINA sshd as ssh client.

!ENTRY org.eclipse.egit.ui 2 0 2019-09-28 10:23:51.002
!MESSAGE Warning: The environment variable HOME is not set. The following directory will be used to store the Git
user global configuration and to define the default location to store repositories: 'C:\Users\ITAUser'. If this is
not correct please set the HOME environment variable and restart Eclipse. Otherwise Git for Windows and
EGit might behave differently since they see different configuration options.
This warning can be switched off on the Team > Git > Confirmations and Warnings preference page.

!ENTRY org.brainwy.liclipsetext.shared_core 4 4 2019-09-28 10:23:52.620
!MESSAGE C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\com.brainwy.liclipse.editor_5.3.0.201908230902\etc\a767gwy0xncwwz0slpfps4tb3 (The system cannot find the path specified)
!STACK 0
java.io.FileNotFoundException: C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\com.brainwy.liclipse.editor_5.3.0.201908230902\etc\a767gwy0xncwwz0slpfps4tb3 (The system cannot find the path specified)
	at java.io.FileOutputStream.open0(Native Method)
	at java.io.FileOutputStream.open(FileOutputStream.java:270)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:213)
	at java.io.FileOutputStream.<init>(FileOutputStream.java:162)
	at org.brainwy.liclipsetext.shared_core.io.FileUtils.writeBytesToFile(FileUtils.java:140)
	at org.brainwy.liclipsetext.shared_core.io.FileUtils.writeStrToFile(FileUtils.java:133)
	at com.brainwy.liclipse.editor.lic.CheckRegistered.run(CheckRegistered.java:146)

!ENTRY org.python.pydev.shared_core 1 1 2019-09-28 10:26:01.428
!MESSAGE - Ok, file is non-null. Getting info on:C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe
- Beggining task:Getting libs totalWork:100
- Setting task name:Mounting executable string...
- Setting task name:Executing:  C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making pythonpath environment... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making exec... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Reading output...
- Setting task name:Waiting for process to finish.
- Setting task name:Process finished.
- Success getting the info. Result:<xml>
<name>python</name>
<version>3.7</version>
<executable>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe</executable>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\DLLs</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib\site-packages</lib>
<forced_lib>Image</forced_lib>
<forced_lib>OpenGL</forced_lib>
<forced_lib>_abc</forced_lib>
<forced_lib>_ast</forced_lib>
<forced_lib>_bisect</forced_lib>
<forced_lib>_blake2</forced_lib>
<forced_lib>_bytesio</forced_lib>
<forced_lib>_codecs</forced_lib>
<forced_lib>_codecs_cn</forced_lib>
<forced_lib>_codecs_hk</forced_lib>
<forced_lib>_codecs_iso2022</forced_lib>
<forced_lib>_codecs_jp</forced_lib>
<forced_lib>_codecs_kr</forced_lib>
<forced_lib>_codecs_tw</forced_lib>
<forced_lib>_collections</forced_lib>
<forced_lib>_contextvars</forced_lib>
<forced_lib>_csv</forced_lib>
<forced_lib>_datetime</forced_lib>
<forced_lib>_fileio</forced_lib>
<forced_lib>_functools</forced_lib>
<forced_lib>_heapq</forced_lib>
<forced_lib>_hotshot</forced_lib>
<forced_lib>_imp</forced_lib>
<forced_lib>_io</forced_lib>
<forced_lib>_json</forced_lib>
<forced_lib>_locale</forced_lib>
<forced_lib>_lsprof</forced_lib>
<forced_lib>_md5</forced_lib>
<forced_lib>_multibytecodec</forced_lib>
<forced_lib>_opcode</forced_lib>
<forced_lib>_operator</forced_lib>
<forced_lib>_pickle</forced_lib>
<forced_lib>_random</forced_lib>
<forced_lib>_sha</forced_lib>
<forced_lib>_sha1</forced_lib>
<forced_lib>_sha256</forced_lib>
<forced_lib>_sha3</forced_lib>
<forced_lib>_sha512</forced_lib>
<forced_lib>_signal</forced_lib>
<forced_lib>_sre</forced_lib>
<forced_lib>_stat</forced_lib>
<forced_lib>_string</forced_lib>
<forced_lib>_struct</forced_lib>
<forced_lib>_subprocess</forced_lib>
<forced_lib>_symtable</forced_lib>
<forced_lib>_thread</forced_lib>
<forced_lib>_tracemalloc</forced_lib>
<forced_lib>_warnings</forced_lib>
<forced_lib>_weakref</forced_lib>
<forced_lib>_winapi</forced_lib>
<forced_lib>_winreg</forced_lib>
<forced_lib>array</forced_lib>
<forced_lib>atexit</forced_lib>
<forced_lib>audioop</forced_lib>
<forced_lib>binascii</forced_lib>
<forced_lib>builtins</forced_lib>
<forced_lib>cPickle</forced_lib>
<forced_lib>cStringIO</forced_lib>
<forced_lib>cmath</forced_lib>
<forced_lib>cv2</forced_lib>
<forced_lib>datetime</forced_lib>
<forced_lib>email</forced_lib>
<forced_lib>errno</forced_lib>
<forced_lib>exceptions</forced_lib>
<forced_lib>faulthandler</forced_lib>
<forced_lib>future_builtins</forced_lib>
<forced_lib>gc</forced_lib>
<forced_lib>gi</forced_lib>
<forced_lib>hashlib</forced_lib>
<forced_lib>imageop</forced_lib>
<forced_lib>imp</forced_lib>
<forced_lib>itertools</forced_lib>
<forced_lib>marshal</forced_lib>
<forced_lib>math</forced_lib>
<forced_lib>mmap</forced_lib>
<forced_lib>mock</forced_lib>
<forced_lib>msvcrt</forced_lib>
<forced_lib>multiprocessing</forced_lib>
<forced_lib>mutagen</forced_lib>
<forced_lib>nt</forced_lib>
<forced_lib>numpy</forced_lib>
<forced_lib>operator</forced_lib>
<forced_lib>os</forced_lib>
<forced_lib>os.path</forced_lib>
<forced_lib>parser</forced_lib>
<forced_lib>pytest</forced_lib>
<forced_lib>scipy</forced_lib>
<forced_lib>signal</forced_lib>
<forced_lib>six</forced_lib>
<forced_lib>socket</forced_lib>
<forced_lib>ssl</forced_lib>
<forced_lib>strop</forced_lib>
<forced_lib>sys</forced_lib>
<forced_lib>thread</forced_lib>
<forced_lib>time</forced_lib>
<forced_lib>winreg</forced_lib>
<forced_lib>wx</forced_lib>
<forced_lib>wxPython</forced_lib>
<forced_lib>xxsubtype</forced_lib>
<forced_lib>zipimport</forced_lib>
<forced_lib>zlib</forced_lib>
<string_substitution_var><key>PY</key><value>37</value></string_substitution_var>
</xml>
- Ok, file is non-null. Getting info on:C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe
- Beggining task:Getting libs totalWork:100
- Setting task name:Mounting executable string...
- Setting task name:Executing:  C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making pythonpath environment... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making exec... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Reading output...
- Setting task name:Waiting for process to finish.
- Setting task name:Process finished.
- Success getting the info. Result:<xml>
<name>python</name>
<version>3.7</version>
<executable>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe</executable>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\DLLs</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib\site-packages</lib>
<forced_lib>Image</forced_lib>
<forced_lib>OpenGL</forced_lib>
<forced_lib>_abc</forced_lib>
<forced_lib>_ast</forced_lib>
<forced_lib>_bisect</forced_lib>
<forced_lib>_blake2</forced_lib>
<forced_lib>_bytesio</forced_lib>
<forced_lib>_codecs</forced_lib>
<forced_lib>_codecs_cn</forced_lib>
<forced_lib>_codecs_hk</forced_lib>
<forced_lib>_codecs_iso2022</forced_lib>
<forced_lib>_codecs_jp</forced_lib>
<forced_lib>_codecs_kr</forced_lib>
<forced_lib>_codecs_tw</forced_lib>
<forced_lib>_collections</forced_lib>
<forced_lib>_contextvars</forced_lib>
<forced_lib>_csv</forced_lib>
<forced_lib>_datetime</forced_lib>
<forced_lib>_fileio</forced_lib>
<forced_lib>_functools</forced_lib>
<forced_lib>_heapq</forced_lib>
<forced_lib>_hotshot</forced_lib>
<forced_lib>_imp</forced_lib>
<forced_lib>_io</forced_lib>
<forced_lib>_json</forced_lib>
<forced_lib>_locale</forced_lib>
<forced_lib>_lsprof</forced_lib>
<forced_lib>_md5</forced_lib>
<forced_lib>_multibytecodec</forced_lib>
<forced_lib>_opcode</forced_lib>
<forced_lib>_operator</forced_lib>
<forced_lib>_pickle</forced_lib>
<forced_lib>_random</forced_lib>
<forced_lib>_sha</forced_lib>
<forced_lib>_sha1</forced_lib>
<forced_lib>_sha256</forced_lib>
<forced_lib>_sha3</forced_lib>
<forced_lib>_sha512</forced_lib>
<forced_lib>_signal</forced_lib>
<forced_lib>_sre</forced_lib>
<forced_lib>_stat</forced_lib>
<forced_lib>_string</forced_lib>
<forced_lib>_struct</forced_lib>
<forced_lib>_subprocess</forced_lib>
<forced_lib>_symtable</forced_lib>
<forced_lib>_thread</forced_lib>
<forced_lib>_tracemalloc</forced_lib>
<forced_lib>_warnings</forced_lib>
<forced_lib>_weakref</forced_lib>
<forced_lib>_winapi</forced_lib>
<forced_lib>_winreg</forced_lib>
<forced_lib>array</forced_lib>
<forced_lib>atexit</forced_lib>
<forced_lib>audioop</forced_lib>
<forced_lib>binascii</forced_lib>
<forced_lib>builtins</forced_lib>
<forced_lib>cPickle</forced_lib>
<forced_lib>cStringIO</forced_lib>
<forced_lib>cmath</forced_lib>
<forced_lib>cv2</forced_lib>
<forced_lib>datetime</forced_lib>
<forced_lib>email</forced_lib>
<forced_lib>errno</forced_lib>
<forced_lib>exceptions</forced_lib>
<forced_lib>faulthandler</forced_lib>
<forced_lib>future_builtins</forced_lib>
<forced_lib>gc</forced_lib>
<forced_lib>gi</forced_lib>
<forced_lib>hashlib</forced_lib>
<forced_lib>imageop</forced_lib>
<forced_lib>imp</forced_lib>
<forced_lib>itertools</forced_lib>
<forced_lib>marshal</forced_lib>
<forced_lib>math</forced_lib>
<forced_lib>mmap</forced_lib>
<forced_lib>mock</forced_lib>
<forced_lib>msvcrt</forced_lib>
<forced_lib>multiprocessing</forced_lib>
<forced_lib>mutagen</forced_lib>
<forced_lib>nt</forced_lib>
<forced_lib>numpy</forced_lib>
<forced_lib>operator</forced_lib>
<forced_lib>os</forced_lib>
<forced_lib>os.path</forced_lib>
<forced_lib>parser</forced_lib>
<forced_lib>pytest</forced_lib>
<forced_lib>scipy</forced_lib>
<forced_lib>signal</forced_lib>
<forced_lib>six</forced_lib>
<forced_lib>socket</forced_lib>
<forced_lib>ssl</forced_lib>
<forced_lib>strop</forced_lib>
<forced_lib>sys</forced_lib>
<forced_lib>thread</forced_lib>
<forced_lib>time</forced_lib>
<forced_lib>winreg</forced_lib>
<forced_lib>wx</forced_lib>
<forced_lib>wxPython</forced_lib>
<forced_lib>xxsubtype</forced_lib>
<forced_lib>zipimport</forced_lib>
<forced_lib>zlib</forced_lib>
<string_substitution_var><key>PY</key><value>37</value></string_substitution_var>
</xml>

!STACK 0
java.lang.RuntimeException: - Ok, file is non-null. Getting info on:C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe
- Beggining task:Getting libs totalWork:100
- Setting task name:Mounting executable string...
- Setting task name:Executing:  C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making pythonpath environment... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making exec... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Reading output...
- Setting task name:Waiting for process to finish.
- Setting task name:Process finished.
- Success getting the info. Result:<xml>
<name>python</name>
<version>3.7</version>
<executable>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe</executable>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\DLLs</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib\site-packages</lib>
<forced_lib>Image</forced_lib>
<forced_lib>OpenGL</forced_lib>
<forced_lib>_abc</forced_lib>
<forced_lib>_ast</forced_lib>
<forced_lib>_bisect</forced_lib>
<forced_lib>_blake2</forced_lib>
<forced_lib>_bytesio</forced_lib>
<forced_lib>_codecs</forced_lib>
<forced_lib>_codecs_cn</forced_lib>
<forced_lib>_codecs_hk</forced_lib>
<forced_lib>_codecs_iso2022</forced_lib>
<forced_lib>_codecs_jp</forced_lib>
<forced_lib>_codecs_kr</forced_lib>
<forced_lib>_codecs_tw</forced_lib>
<forced_lib>_collections</forced_lib>
<forced_lib>_contextvars</forced_lib>
<forced_lib>_csv</forced_lib>
<forced_lib>_datetime</forced_lib>
<forced_lib>_fileio</forced_lib>
<forced_lib>_functools</forced_lib>
<forced_lib>_heapq</forced_lib>
<forced_lib>_hotshot</forced_lib>
<forced_lib>_imp</forced_lib>
<forced_lib>_io</forced_lib>
<forced_lib>_json</forced_lib>
<forced_lib>_locale</forced_lib>
<forced_lib>_lsprof</forced_lib>
<forced_lib>_md5</forced_lib>
<forced_lib>_multibytecodec</forced_lib>
<forced_lib>_opcode</forced_lib>
<forced_lib>_operator</forced_lib>
<forced_lib>_pickle</forced_lib>
<forced_lib>_random</forced_lib>
<forced_lib>_sha</forced_lib>
<forced_lib>_sha1</forced_lib>
<forced_lib>_sha256</forced_lib>
<forced_lib>_sha3</forced_lib>
<forced_lib>_sha512</forced_lib>
<forced_lib>_signal</forced_lib>
<forced_lib>_sre</forced_lib>
<forced_lib>_stat</forced_lib>
<forced_lib>_string</forced_lib>
<forced_lib>_struct</forced_lib>
<forced_lib>_subprocess</forced_lib>
<forced_lib>_symtable</forced_lib>
<forced_lib>_thread</forced_lib>
<forced_lib>_tracemalloc</forced_lib>
<forced_lib>_warnings</forced_lib>
<forced_lib>_weakref</forced_lib>
<forced_lib>_winapi</forced_lib>
<forced_lib>_winreg</forced_lib>
<forced_lib>array</forced_lib>
<forced_lib>atexit</forced_lib>
<forced_lib>audioop</forced_lib>
<forced_lib>binascii</forced_lib>
<forced_lib>builtins</forced_lib>
<forced_lib>cPickle</forced_lib>
<forced_lib>cStringIO</forced_lib>
<forced_lib>cmath</forced_lib>
<forced_lib>cv2</forced_lib>
<forced_lib>datetime</forced_lib>
<forced_lib>email</forced_lib>
<forced_lib>errno</forced_lib>
<forced_lib>exceptions</forced_lib>
<forced_lib>faulthandler</forced_lib>
<forced_lib>future_builtins</forced_lib>
<forced_lib>gc</forced_lib>
<forced_lib>gi</forced_lib>
<forced_lib>hashlib</forced_lib>
<forced_lib>imageop</forced_lib>
<forced_lib>imp</forced_lib>
<forced_lib>itertools</forced_lib>
<forced_lib>marshal</forced_lib>
<forced_lib>math</forced_lib>
<forced_lib>mmap</forced_lib>
<forced_lib>mock</forced_lib>
<forced_lib>msvcrt</forced_lib>
<forced_lib>multiprocessing</forced_lib>
<forced_lib>mutagen</forced_lib>
<forced_lib>nt</forced_lib>
<forced_lib>numpy</forced_lib>
<forced_lib>operator</forced_lib>
<forced_lib>os</forced_lib>
<forced_lib>os.path</forced_lib>
<forced_lib>parser</forced_lib>
<forced_lib>pytest</forced_lib>
<forced_lib>scipy</forced_lib>
<forced_lib>signal</forced_lib>
<forced_lib>six</forced_lib>
<forced_lib>socket</forced_lib>
<forced_lib>ssl</forced_lib>
<forced_lib>strop</forced_lib>
<forced_lib>sys</forced_lib>
<forced_lib>thread</forced_lib>
<forced_lib>time</forced_lib>
<forced_lib>winreg</forced_lib>
<forced_lib>wx</forced_lib>
<forced_lib>wxPython</forced_lib>
<forced_lib>xxsubtype</forced_lib>
<forced_lib>zipimport</forced_lib>
<forced_lib>zlib</forced_lib>
<string_substitution_var><key>PY</key><value>37</value></string_substitution_var>
</xml>
- Ok, file is non-null. Getting info on:C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe
- Beggining task:Getting libs totalWork:100
- Setting task name:Mounting executable string...
- Setting task name:Executing:  C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making pythonpath environment... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Making exec... C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe -u "C:\Program Files\Brainwy\LiClipse 5.3.0\plugins\org.python.pydev.core_7.3.0.201908230902\pysrc\interpreterInfo.py"
- Setting task name:Reading output...
- Setting task name:Waiting for process to finish.
- Setting task name:Process finished.
- Success getting the info. Result:<xml>
<name>python</name>
<version>3.7</version>
<executable>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe</executable>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\DLLs</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32</lib>
<lib>C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\lib\site-packages</lib>
<forced_lib>Image</forced_lib>
<forced_lib>OpenGL</forced_lib>
<forced_lib>_abc</forced_lib>
<forced_lib>_ast</forced_lib>
<forced_lib>_bisect</forced_lib>
<forced_lib>_blake2</forced_lib>
<forced_lib>_bytesio</forced_lib>
<forced_lib>_codecs</forced_lib>
<forced_lib>_codecs_cn</forced_lib>
<forced_lib>_codecs_hk</forced_lib>
<forced_lib>_codecs_iso2022</forced_lib>
<forced_lib>_codecs_jp</forced_lib>
<forced_lib>_codecs_kr</forced_lib>
<forced_lib>_codecs_tw</forced_lib>
<forced_lib>_collections</forced_lib>
<forced_lib>_contextvars</forced_lib>
<forced_lib>_csv</forced_lib>
<forced_lib>_datetime</forced_lib>
<forced_lib>_fileio</forced_lib>
<forced_lib>_functools</forced_lib>
<forced_lib>_heapq</forced_lib>
<forced_lib>_hotshot</forced_lib>
<forced_lib>_imp</forced_lib>
<forced_lib>_io</forced_lib>
<forced_lib>_json</forced_lib>
<forced_lib>_locale</forced_lib>
<forced_lib>_lsprof</forced_lib>
<forced_lib>_md5</forced_lib>
<forced_lib>_multibytecodec</forced_lib>
<forced_lib>_opcode</forced_lib>
<forced_lib>_operator</forced_lib>
<forced_lib>_pickle</forced_lib>
<forced_lib>_random</forced_lib>
<forced_lib>_sha</forced_lib>
<forced_lib>_sha1</forced_lib>
<forced_lib>_sha256</forced_lib>
<forced_lib>_sha3</forced_lib>
<forced_lib>_sha512</forced_lib>
<forced_lib>_signal</forced_lib>
<forced_lib>_sre</forced_lib>
<forced_lib>_stat</forced_lib>
<forced_lib>_string</forced_lib>
<forced_lib>_struct</forced_lib>
<forced_lib>_subprocess</forced_lib>
<forced_lib>_symtable</forced_lib>
<forced_lib>_thread</forced_lib>
<forced_lib>_tracemalloc</forced_lib>
<forced_lib>_warnings</forced_lib>
<forced_lib>_weakref</forced_lib>
<forced_lib>_winapi</forced_lib>
<forced_lib>_winreg</forced_lib>
<forced_lib>array</forced_lib>
<forced_lib>atexit</forced_lib>
<forced_lib>audioop</forced_lib>
<forced_lib>binascii</forced_lib>
<forced_lib>builtins</forced_lib>
<forced_lib>cPickle</forced_lib>
<forced_lib>cStringIO</forced_lib>
<forced_lib>cmath</forced_lib>
<forced_lib>cv2</forced_lib>
<forced_lib>datetime</forced_lib>
<forced_lib>email</forced_lib>
<forced_lib>errno</forced_lib>
<forced_lib>exceptions</forced_lib>
<forced_lib>faulthandler</forced_lib>
<forced_lib>future_builtins</forced_lib>
<forced_lib>gc</forced_lib>
<forced_lib>gi</forced_lib>
<forced_lib>hashlib</forced_lib>
<forced_lib>imageop</forced_lib>
<forced_lib>imp</forced_lib>
<forced_lib>itertools</forced_lib>
<forced_lib>marshal</forced_lib>
<forced_lib>math</forced_lib>
<forced_lib>mmap</forced_lib>
<forced_lib>mock</forced_lib>
<forced_lib>msvcrt</forced_lib>
<forced_lib>multiprocessing</forced_lib>
<forced_lib>mutagen</forced_lib>
<forced_lib>nt</forced_lib>
<forced_lib>numpy</forced_lib>
<forced_lib>operator</forced_lib>
<forced_lib>os</forced_lib>
<forced_lib>os.path</forced_lib>
<forced_lib>parser</forced_lib>
<forced_lib>pytest</forced_lib>
<forced_lib>scipy</forced_lib>
<forced_lib>signal</forced_lib>
<forced_lib>six</forced_lib>
<forced_lib>socket</forced_lib>
<forced_lib>ssl</forced_lib>
<forced_lib>strop</forced_lib>
<forced_lib>sys</forced_lib>
<forced_lib>thread</forced_lib>
<forced_lib>time</forced_lib>
<forced_lib>winreg</forced_lib>
<forced_lib>wx</forced_lib>
<forced_lib>wxPython</forced_lib>
<forced_lib>xxsubtype</forced_lib>
<forced_lib>zipimport</forced_lib>
<forced_lib>zlib</forced_lib>
<string_substitution_var><key>PY</key><value>37</value></string_substitution_var>
</xml>

	at org.python.pydev.core.log.Log.logInfo(Log.java:45)
	at org.python.pydev.ui.pythonpathconf.AutoConfigMaker.applyOperation(AutoConfigMaker.java:191)
	at org.python.pydev.ui.pythonpathconf.AutoConfigMaker.autoConfigSingleApply(AutoConfigMaker.java:128)
	at org.python.pydev.ui.PyProjectPythonDetails$ProjectInterpreterAndGrammarConfig$2.widgetSelected(PyProjectPythonDetails.java:326)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:252)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4131)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1055)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1079)
	at org.eclipse.swt.widgets.Widget.sendSelectionEvent(Widget.java:1096)
	at org.eclipse.swt.widgets.Link.WM_LBUTTONUP(Link.java:1013)
	at org.eclipse.swt.widgets.Control.windowProc(Control.java:4805)
	at org.eclipse.swt.widgets.Display.windowProc(Display.java:4770)
	at org.eclipse.swt.internal.win32.OS.DispatchMessage(Native Method)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3545)
	at org.eclipse.jface.window.Window.runEventLoop(Window.java:823)
	at org.eclipse.jface.window.Window.open(Window.java:799)
	at org.eclipse.ui.internal.actions.NewWizardShortcutAction.run(NewWizardShortcutAction.java:136)
	at org.eclipse.jface.action.Action.runWithEvent(Action.java:474)
	at org.eclipse.jface.action.ActionContributionItem.handleWidgetSelection(ActionContributionItem.java:568)
	at org.eclipse.jface.action.ActionContributionItem.lambda$4(ActionContributionItem.java:400)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4131)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1055)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:3944)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3547)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1173)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1062)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:566)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:150)
	at com.brainwy.liclipse.rcp.IDEApplication.start(IDEApplication.java:236)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:137)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:107)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:400)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:661)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:597)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1476)

!ENTRY org.python.pydev.shared_core 1 1 2019-09-28 10:26:01.435
!MESSAGE Restoring info for: C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe
!STACK 0
java.lang.RuntimeException: Restoring info for: C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.getInterpretersFromPersistedString(AbstractInterpreterManager.java:453)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.internalRecreateCacheGetInterpreterInfos(AbstractInterpreterManager.java:279)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:576)
	at org.python.pydev.ui.pythonpathconf.AutoConfigMaker$1.run(AutoConfigMaker.java:152)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)
Caused by: java.io.IOException: Expecting: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.python.pydev\v1_dpqicwzfemxm4z6evvl4oqjwk\modulesKeys to exist (and be a file).
	at org.python.pydev.ast.codecompletion.revisited.ModulesManager.loadFromFile(ModulesManager.java:296)
	at org.python.pydev.ast.codecompletion.revisited.SystemModulesManager.load(SystemModulesManager.java:400)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.getInterpretersFromPersistedString(AbstractInterpreterManager.java:451)
	... 4 more

!ENTRY org.python.pydev.shared_core 1 1 2019-09-28 10:26:04.320
!MESSAGE Finished restoring information for: C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe at: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.python.pydev\v1_dpqicwzfemxm4z6evvl4oqjwk
!STACK 0
java.lang.RuntimeException: Finished restoring information for: C:\Users\ITAUser\AppData\Local\Programs\Python\Python37-32\python.exe at: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.python.pydev\v1_dpqicwzfemxm4z6evvl4oqjwk
	at org.python.pydev.core.log.Log.logInfo(Log.java:45)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.getInterpretersFromPersistedString(AbstractInterpreterManager.java:498)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.internalRecreateCacheGetInterpreterInfos(AbstractInterpreterManager.java:279)
	at org.python.pydev.ast.interpreter_managers.AbstractInterpreterManager.setInfos(AbstractInterpreterManager.java:576)
	at org.python.pydev.ui.pythonpathconf.AutoConfigMaker$1.run(AutoConfigMaker.java:152)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.python.pydev.shared_core 4 4 2019-09-28 10:26:05.520
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\com.python.pydev.analysis\python_v1_dpqicwzfemxm4z6evvl4oqjwk\python.pydevsysteminfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-09-28 10:26:13.826
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\variables\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-09-28 12:00:30.817
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-05 10:30:41.056
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathOperators\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-05 10:50:41.709
!MESSAGE Error. returning: 0
!STACK 0
java.nio.file.NoSuchFileException: C:\Users\ITAUser\Desktop\Python\mathematicalOperations\practiceMatematicalOperations.py
	at sun.nio.fs.WindowsException.translateToIOException(WindowsException.java:79)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:97)
	at sun.nio.fs.WindowsException.rethrowAsIOException(WindowsException.java:102)
	at sun.nio.fs.WindowsFileAttributeViews$Basic.readAttributes(WindowsFileAttributeViews.java:53)
	at sun.nio.fs.WindowsFileAttributeViews$Basic.readAttributes(WindowsFileAttributeViews.java:38)
	at sun.nio.fs.WindowsFileSystemProvider.readAttributes(WindowsFileSystemProvider.java:193)
	at java.nio.file.Files.readAttributes(Files.java:1737)
	at java.nio.file.Files.getLastModifiedTime(Files.java:2266)
	at org.python.pydev.shared_core.io.FileUtils.lastModified(FileUtils.java:1081)
	at org.python.pydev.shared_core.io.FileUtils.lastModified(FileUtils.java:1072)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalDependencyInfo.addInfoToModuleOnRestoreInsertCommand(AbstractAdditionalDependencyInfo.java:529)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.processInsert(AbstractAdditionalInfoWithBuild.java:192)
	at org.python.pydev.core.DeltaSaver$DeltaInsertCommand.processWith(DeltaSaver.java:98)
	at org.python.pydev.core.DeltaSaver.processDeltas(DeltaSaver.java:318)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalInfoWithBuild.restoreSavedInfo(AbstractAdditionalInfoWithBuild.java:103)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalDependencyInfo.loadContentsFromFile(AbstractAdditionalDependencyInfo.java:499)
	at com.python.pydev.analysis.additionalinfo.AbstractAdditionalDependencyInfo.load(AbstractAdditionalDependencyInfo.java:414)
	at com.python.pydev.analysis.additionalinfo.AdditionalProjectInterpreterInfo.getAdditionalInfoForProject(AdditionalProjectInterpreterInfo.java:247)
	at com.python.pydev.analysis.additionalinfo.AdditionalProjectInterpreterInfo.recreateAllInfo(AdditionalProjectInterpreterInfo.java:330)
	at com.python.pydev.analysis.additionalinfo.builders.InterpreterObserver.notifyProjectPythonpathRestored(InterpreterObserver.java:41)
	at org.python.pydev.plugin.nature.PythonNature$RebuildPythonNatureModules.run(PythonNature.java:200)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:63)

!ENTRY org.eclipse.ltk.ui.refactoring 4 10000 2019-10-05 10:50:41.961
!MESSAGE Internal Error
!STACK 1
org.eclipse.core.internal.resources.ResourceException: Problems encountered while moving resources.
	at org.eclipse.core.internal.resources.Resource.move(Resource.java:1504)
	at org.eclipse.ltk.core.refactoring.resource.RenameResourceChange.perform(RenameResourceChange.java:118)
	at org.eclipse.ltk.core.refactoring.CompositeChange.perform(CompositeChange.java:283)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation$1.run(PerformChangeOperation.java:260)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.executeChange(PerformChangeOperation.java:308)
	at org.eclipse.ltk.internal.ui.refactoring.UIPerformChangeOperation.executeChange(UIPerformChangeOperation.java:99)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.run(PerformChangeOperation.java:219)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.internal.ui.refactoring.WorkbenchRunnableAdapter.run(WorkbenchRunnableAdapter.java:89)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)
Contains: Error moving metadata area from C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations to C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathOperator.
org.eclipse.core.runtime.CoreException: Could not move: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations.
	at org.eclipse.core.internal.filesystem.Policy.error(Policy.java:48)
	at org.eclipse.core.filesystem.provider.FileStore.move(FileStore.java:392)
	at org.eclipse.core.internal.filesystem.local.LocalFile.move(LocalFile.java:395)
	at org.eclipse.core.internal.resources.ResourceTree.movedProjectSubtree(ResourceTree.java:672)
	at org.eclipse.core.internal.resources.ResourceTree.standardMoveProject(ResourceTree.java:1104)
	at org.eclipse.core.internal.resources.Resource.unprotectedMove(Resource.java:1814)
	at org.eclipse.core.internal.resources.Resource.move(Resource.java:1492)
	at org.eclipse.ltk.core.refactoring.resource.RenameResourceChange.perform(RenameResourceChange.java:118)
	at org.eclipse.ltk.core.refactoring.CompositeChange.perform(CompositeChange.java:283)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation$1.run(PerformChangeOperation.java:260)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.executeChange(PerformChangeOperation.java:308)
	at org.eclipse.ltk.internal.ui.refactoring.UIPerformChangeOperation.executeChange(UIPerformChangeOperation.java:99)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.run(PerformChangeOperation.java:219)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.internal.ui.refactoring.WorkbenchRunnableAdapter.run(WorkbenchRunnableAdapter.java:89)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)
Caused by: org.eclipse.core.runtime.CoreException: Could not read from source when writing file: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations\com.python.pydev.analysis\v2_indexcache\lcL6dot1\write.lock
	at org.eclipse.core.internal.filesystem.Policy.error(Policy.java:48)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:68)
	at org.eclipse.core.filesystem.provider.FileStore.copyFile(FileStore.java:202)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:131)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.move(FileStore.java:387)
	... 17 more
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(FileInputStream.java:233)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:65)
	... 33 more
!SUBENTRY 1 org.eclipse.core.resources 4 4 2019-10-05 10:50:41.964
!MESSAGE Problems encountered while moving resources.
!SUBENTRY 2 org.eclipse.core.resources 4 568 2019-10-05 10:50:41.964
!MESSAGE Error moving metadata area from C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations to C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathOperator.
!STACK 1
org.eclipse.core.runtime.CoreException: Could not move: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations.
	at org.eclipse.core.internal.filesystem.Policy.error(Policy.java:48)
	at org.eclipse.core.filesystem.provider.FileStore.move(FileStore.java:392)
	at org.eclipse.core.internal.filesystem.local.LocalFile.move(LocalFile.java:395)
	at org.eclipse.core.internal.resources.ResourceTree.movedProjectSubtree(ResourceTree.java:672)
	at org.eclipse.core.internal.resources.ResourceTree.standardMoveProject(ResourceTree.java:1104)
	at org.eclipse.core.internal.resources.Resource.unprotectedMove(Resource.java:1814)
	at org.eclipse.core.internal.resources.Resource.move(Resource.java:1492)
	at org.eclipse.ltk.core.refactoring.resource.RenameResourceChange.perform(RenameResourceChange.java:118)
	at org.eclipse.ltk.core.refactoring.CompositeChange.perform(CompositeChange.java:283)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation$1.run(PerformChangeOperation.java:260)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.executeChange(PerformChangeOperation.java:308)
	at org.eclipse.ltk.internal.ui.refactoring.UIPerformChangeOperation.executeChange(UIPerformChangeOperation.java:99)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.run(PerformChangeOperation.java:219)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.internal.ui.refactoring.WorkbenchRunnableAdapter.run(WorkbenchRunnableAdapter.java:89)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)
Caused by: org.eclipse.core.runtime.CoreException: Could not read from source when writing file: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations\com.python.pydev.analysis\v2_indexcache\lcL6dot1\write.lock
	at org.eclipse.core.internal.filesystem.Policy.error(Policy.java:48)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:68)
	at org.eclipse.core.filesystem.provider.FileStore.copyFile(FileStore.java:202)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:131)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.move(FileStore.java:387)
	... 17 more
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(FileInputStream.java:233)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:65)
	... 33 more
!SUBENTRY 3 org.eclipse.core.filesystem 4 272 2019-10-05 10:50:41.964
!MESSAGE Could not move: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations.
!STACK 1
org.eclipse.core.runtime.CoreException: Could not read from source when writing file: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations\com.python.pydev.analysis\v2_indexcache\lcL6dot1\write.lock
	at org.eclipse.core.internal.filesystem.Policy.error(Policy.java:48)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:68)
	at org.eclipse.core.filesystem.provider.FileStore.copyFile(FileStore.java:202)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:131)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.move(FileStore.java:387)
	at org.eclipse.core.internal.filesystem.local.LocalFile.move(LocalFile.java:395)
	at org.eclipse.core.internal.resources.ResourceTree.movedProjectSubtree(ResourceTree.java:672)
	at org.eclipse.core.internal.resources.ResourceTree.standardMoveProject(ResourceTree.java:1104)
	at org.eclipse.core.internal.resources.Resource.unprotectedMove(Resource.java:1814)
	at org.eclipse.core.internal.resources.Resource.move(Resource.java:1492)
	at org.eclipse.ltk.core.refactoring.resource.RenameResourceChange.perform(RenameResourceChange.java:118)
	at org.eclipse.ltk.core.refactoring.CompositeChange.perform(CompositeChange.java:283)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation$1.run(PerformChangeOperation.java:260)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.executeChange(PerformChangeOperation.java:308)
	at org.eclipse.ltk.internal.ui.refactoring.UIPerformChangeOperation.executeChange(UIPerformChangeOperation.java:99)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.run(PerformChangeOperation.java:219)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.internal.ui.refactoring.WorkbenchRunnableAdapter.run(WorkbenchRunnableAdapter.java:89)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(FileInputStream.java:233)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:65)
	... 33 more
!SUBENTRY 4 org.eclipse.core.filesystem 4 271 2019-10-05 10:50:41.964
!MESSAGE Could not read from source when writing file: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\mathematicalOperations\com.python.pydev.analysis\v2_indexcache\lcL6dot1\write.lock
!STACK 0
java.io.IOException: The process cannot access the file because another process has locked a portion of the file
	at java.io.FileInputStream.readBytes(Native Method)
	at java.io.FileInputStream.read(FileInputStream.java:233)
	at org.eclipse.core.filesystem.provider.FileStore.transferStreams(FileStore.java:65)
	at org.eclipse.core.filesystem.provider.FileStore.copyFile(FileStore.java:202)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:131)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.copyDirectory(FileStore.java:169)
	at org.eclipse.core.filesystem.provider.FileStore.copy(FileStore.java:129)
	at org.eclipse.core.internal.filesystem.local.LocalFile.copy(LocalFile.java:127)
	at org.eclipse.core.filesystem.provider.FileStore.move(FileStore.java:387)
	at org.eclipse.core.internal.filesystem.local.LocalFile.move(LocalFile.java:395)
	at org.eclipse.core.internal.resources.ResourceTree.movedProjectSubtree(ResourceTree.java:672)
	at org.eclipse.core.internal.resources.ResourceTree.standardMoveProject(ResourceTree.java:1104)
	at org.eclipse.core.internal.resources.Resource.unprotectedMove(Resource.java:1814)
	at org.eclipse.core.internal.resources.Resource.move(Resource.java:1492)
	at org.eclipse.ltk.core.refactoring.resource.RenameResourceChange.perform(RenameResourceChange.java:118)
	at org.eclipse.ltk.core.refactoring.CompositeChange.perform(CompositeChange.java:283)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation$1.run(PerformChangeOperation.java:260)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.executeChange(PerformChangeOperation.java:308)
	at org.eclipse.ltk.internal.ui.refactoring.UIPerformChangeOperation.executeChange(UIPerformChangeOperation.java:99)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.run(PerformChangeOperation.java:219)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.internal.ui.refactoring.WorkbenchRunnableAdapter.run(WorkbenchRunnableAdapter.java:89)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-05 11:05:23.389
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\comparisonOperators\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-05 12:17:37.060
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\logicOperators\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-19 10:13:52.189
!MESSAGE do not share IndexWriterConfig instances across IndexWriters
!STACK 0
java.lang.IllegalStateException: do not share IndexWriterConfig instances across IndexWriters
	at org.apache.lucene.index.IndexWriterConfig.setIndexWriter(IndexWriterConfig.java:115)
	at org.apache.lucene.index.IndexWriter.<init>(IndexWriter.java:770)
	at org.python.pydev.shared_core.index.IndexApi.init(IndexApi.java:124)
	at org.python.pydev.shared_core.index.IndexApi.<init>(IndexApi.java:99)
	at org.python.pydev.shared_core.index.IndexApi.<init>(IndexApi.java:112)
	at com.python.pydev.analysis.additionalinfo.ReferenceSearchesLucene.internalSearch(ReferenceSearchesLucene.java:157)
	at com.python.pydev.analysis.additionalinfo.ReferenceSearchesLucene.search(ReferenceSearchesLucene.java:94)
	at com.python.pydev.analysis.additionalinfo.AdditionalProjectInterpreterInfo.getModulesWithToken(AdditionalProjectInterpreterInfo.java:408)
	at com.python.pydev.analysis.refactoring.refactorer.RefactorerFindReferences.findPossibleReferences(RefactorerFindReferences.java:123)
	at com.python.pydev.analysis.refactoring.wizards.rename.AbstractRenameRefactorProcess.findFilesWithPossibleReferences(AbstractRenameRefactorProcess.java:304)
	at com.python.pydev.analysis.refactoring.wizards.rename.AbstractRenameWorkspaceRefactorProcess.doCheckInitialOnWorkspace(AbstractRenameWorkspaceRefactorProcess.java:162)
	at com.python.pydev.analysis.refactoring.wizards.rename.PyRenameImportProcess.doCheckInitialOnWorkspace(PyRenameImportProcess.java:171)
	at com.python.pydev.analysis.refactoring.wizards.rename.AbstractRenameWorkspaceRefactorProcess.findReferencesToRenameOnWorkspace(AbstractRenameWorkspaceRefactorProcess.java:133)
	at com.python.pydev.analysis.refactoring.wizards.rename.AbstractRenameRefactorProcess.findReferencesToRename(AbstractRenameRefactorProcess.java:196)
	at com.python.pydev.analysis.refactoring.wizards.rename.PyRenameImportProcess.findReferencesToRename(PyRenameImportProcess.java:72)
	at com.python.pydev.analysis.refactoring.wizards.rename.PyReferenceSearcher.search(PyReferenceSearcher.java:152)
	at com.python.pydev.analysis.refactoring.wizards.rename.PyRenameEntryPoint.checkFinalConditions(PyRenameEntryPoint.java:269)
	at org.eclipse.ltk.core.refactoring.participants.ProcessorBasedRefactoring.checkFinalConditions(ProcessorBasedRefactoring.java:225)
	at org.eclipse.ltk.core.refactoring.CheckConditionsOperation.run(CheckConditionsOperation.java:86)
	at org.eclipse.ltk.core.refactoring.CreateChangeOperation.run(CreateChangeOperation.java:122)
	at org.eclipse.ltk.core.refactoring.PerformChangeOperation.run(PerformChangeOperation.java:210)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2295)
	at org.eclipse.core.internal.resources.Workspace.run(Workspace.java:2322)
	at org.eclipse.ltk.internal.ui.refactoring.WorkbenchRunnableAdapter.run(WorkbenchRunnableAdapter.java:89)
	at org.eclipse.jface.operation.ModalContext$ModalContextThread.run(ModalContext.java:122)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-19 10:24:44.291
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\debuggingComparison\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)

!ENTRY org.python.pydev.shared_core 4 4 2019-10-19 11:38:53.138
!MESSAGE Info: Rebuilding internal caches: C:\Users\ITAUser\Desktop\Python\.metadata\.plugins\org.eclipse.core.resources\.projects\conditionals\com.python.pydev.analysis\AdditionalProjectInterpreterInfo.pydevinfo (Expected error to be provided and got no error!)
